pkg/util/cluster/cluster.go

```diff
@@ -119,6 +120,7 @@ func New(log *logrus.Entry, environment env.Core, ci bool) (*Cluster, error) {
 		openshiftclustersv20220904:        redhatopenshift20220904.NewOpenShiftClustersClient(environment.Environment(), environment.SubscriptionID(), authorizer),
 		securitygroups:                    network.NewSecurityGroupsClient(environment.Environment(), environment.SubscriptionID(), authorizer),
 		subnets:                           network.NewSubnetsClient(environment.Environment(), environment.SubscriptionID(), authorizer),
+		vnets:                             network.NewVirtualNetworksClient(environment.Environment(), environment.SubscriptionID(), authorizer),
```

Comment:
This field is only "used" here (in this case instantiated), so it is really never used. Can we removed for now?

Reply:
Same goes with this one. Will remove it.